templates/resources/application.md.tmpl

```diff
@@ -2,8 +2,8 @@
 
 {{ .Description | trimspace }}
 
-## Limitations
-Axual Terraform Provider only supports Custom Application Type. Connector Application Type will be supported in the future. To use Connector Application Type please use the Self Service UI. Read more: https://docs.axual.io/axual/2023.2/self-service/application-management.html#custom-vs-connector
+## Features
```

Comment:
Let's leave the limitations section and add all the dependencies we keep needing between the different resources. We could also mention the limitation when working with hybrid setups. (Some things don't get imported when edited outside terraform)

Reply:
Fixed.